compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -126,7 +126,7 @@ void addPackBasedPassPipeline(OpPassManager &pm, TilingConfig &tilingConfig) {
   // Promote inputs and result to shared memory
   AMDAIEBufferizeToAllocationOptions bufferizeOptions0;
   bufferizeOptions0.memorySpace = 1;
-  bufferizeOptions0.bufferizeLevel = -1;
+  bufferizeOptions0.bufferizeOperand = BufferizeOperand::InputOutput;
```

Comment:
I thought we werent promoting the outputs at this level. Wont this allocate a memory for the output in shared memory?